apps/src/aichat/views/AichatView.tsx

```diff
@@ -3,38 +3,46 @@
 // This is a React client for an aichat level.
 
 import React from 'react';
-import {useAppDispatch} from '@cdo/apps/util/reduxHooks';
-import Aichat from './Aichat';
-import {
-  sendSuccessReport,
-  navigateToNextLevel,
-} from '@cdo/apps/code-studio/progressRedux';
-import aichatLocale from '../locale';
-import styles from './aichat.module.scss';
 import PanelContainer from '@cdo/apps/lab2/views/components/PanelContainer';
+import ChatMessage from './ChatMessage';
+import UserChatMessageEditor from './UserChatMessageEditor';
+import {demoChatMessages} from './chatMessageShape';
+import moduleStyles from './aichat.module.scss';
+import classNames from 'classnames';
 
 const AichatView: React.FunctionComponent = () => {
-  const dispatch = useAppDispatch();
-
-  const nextButtonPressed = () => {
-    const appType = 'aichat';
-    dispatch(sendSuccessReport(appType));
-    dispatch(navigateToNextLevel());
-  };
-
   return (
-    <div id="aichat">
-      <Aichat>
-        <PanelContainer id="instructions-panel" headerText="Instructions" />
-        <button
-          id="aichat-continue-button"
-          type="button"
-          onClick={() => nextButtonPressed()}
-          className={styles.buttonNext}
-        >
-          {aichatLocale.continue()}
-        </button>
-      </Aichat>
+    <div id="aichat-lab" className={moduleStyles.aichatLab}>
+      <div id="chat-workspace-area">
```

Comment:
Might be good to pull out the "chat-workspace-area" into its own component as well, just to make sure this top level component doesn't get too bloated? that's something we ran into with Music Lab where in the beginning, we added most functionality directly to our top-level view (MusicView). this made the file pretty hard to parse as we added more and more, until we started intentionally removing code from it

Reply:
Sure! I actually had a `chatWorkspace` in this [earlier commit](https://github.com/code-dot-org/code-dot-org/pull/53080/commits/18b0f5c4417e2b93fb7348fa302c10cd7e751bd8) but was trying to fix the CSS `flex` display so will add it back now - thanks!

Reply:
Hey Sanchit - I added `chatWorkspace` back, but now 'flex' doesn't seem to be working as expected. I posted a couple screencast videos in the updated description so removing `chatWorkspace` for now.

Reply:
This is now resolved. I learned that the `flex` styling is not inherited from parent and must include in the div.